lib/Twig/ContainerRuntimeLoader.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Psr\Container\ContainerInterface;
+
+/**
+ * Lazily loads Twig runtime implementations from a container.
```

Comment:
I would say ``from a PSR-11 container``.

And the long description of the class should explain that it expects being able to use the runtime class as an identifier in the container